src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { Address } from 'data/graphql'
+import { OfferData } from 'pages/OfferNew/types'
+import { useTextKeys } from 'utils/textKeys'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { getHouseholdSize } from '../utils'
+
+type Props = {
+  offerData: OfferData
+}
+
+type QuoteWithStreet = {
+  quoteDetails: { street: Address['street'] }
+}
+
+const Container = styled.div`
+  width: 100%;
+  padding: 2rem 0;
+  color: ${colorsV3.white};
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    width: auto;
+    padding: 0;
+  }
+`
+
+const Heading = styled.h1`
+  text-transform: uppercase;
+  font-size: 0.75rem;
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    font-size: 1rem;
+  }
+`
+
+const OfferInfo = styled.div`
+  font-size: 2rem;
+  word-break: break-all;
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    font-size: 3rem;
+  }
+`
+
+export const HeroOfferDetails: React.FC<Props> = ({ offerData }) => {
+  const [numberCoInsured, setNumberCoInsured] = useState<number | null>(null)
+  const [street, setStreet] = useState<Address['street'] | null>(null)
```

Comment:
I see the point of having Address['street'] but might it not be sufficient with string? Just a thought, not strongly opinionated

Reply:
Ah, I see your point, `Address['street']` is equal to a string. Either or works for me, having just string would remove one import :)

Reply:
I wanted it to be clear that it's not just any string. I guess my thinking is that everything that we get from the back-end should also be typed accordingly, with the types generated from the GraphQL schema.

Reply:
So I'll leave it as is 😄